### PR TITLE
week 4 mailroom part2

### PR DESCRIPTION
**Pull request by conwaystern**
Mailroom part 2 program completed.

### CODE REVIEW

File: students/rick_halsell/lesson04/mailroom_part2.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+import os
+import sys
+import operator
+from operator import itemgetter
```
Comment (charlesrobison) [inline students/rick_halsell/lesson04/mailroom_part2.py]: Forgot to import time?

File: students/rick_halsell/lesson04/mailroom_part2.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+# Using switch case for main menu prompt
+switch_main_menu_dict={
+    0: entry_function,
+    1: thank_you_func,
+    2: report_func,
+    3: execute_again_func,
+    4: exit_func}
+
+if __name__ == "__main__":
+    entry_function()
```
Comment (charlesrobison) [inline students/rick_halsell/lesson04/mailroom_part2.py]: Excellent job on this!

File: students/rick_halsell/lesson04/mailroom_part2.py (hunk 1 above)
Comment (conwaystern) [reply students/rick_halsell/lesson04/mailroom_part2.py]: Yes, I added it back to my updated version.

File: students/rick_halsell/lesson04/mailroom_part2.py (hunk 2 above)
Comment (conwaystern) [reply students/rick_halsell/lesson04/mailroom_part2.py]: Thank you!

**Comment by conwaystern** [comment]
Added in the updated mailroom part 2 from lesson04, homework for lesson05, and an updated version of mailroom part3.

File: students/rick_halsell/lesson04/students.txt [hunk 3]
```diff
@@ -0,0 +1,31 @@
+Name: Nickname, languages
+Swift, Taylor: python, java, perl
+Swift, Samuel: Sam
+Brooks, Garth: fortran, java, matlab, bash
+Buble, Michael: python, powershell
+Stefani, Gwen: c#, javascript, python, typescript
+Gaye, Marvin: c++, java
+Jagger, Michael: Mick, shell, python
+Lennon, John:
```
Comment (charlesrobison) [inline students/rick_halsell/lesson04/students.txt]: :)

File: students/rick_halsell/lesson05/mailroom_part3.py [hunk 4]
```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/env python3
+import os
+import sys
+import operator
```
Comment (charlesrobison) [inline students/rick_halsell/lesson05/mailroom_part3.py]: operator imported but not used?

File: students/rick_halsell/lesson05/mailroom_part3.py [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+import time
+from operator import itemgetter
```
Comment (charlesrobison) [inline students/rick_halsell/lesson05/mailroom_part3.py]: Same here

**Pull request merged**